### ISSUE TITLE
I2C_CR2_TXDMAEN undeclared

### ISSUE THREAD

**Issue by jaguarondi** [MaJerle/stm32fxxx_hal_libraries#17]
Just noticed that I2C_CR2_TXDMAEN doesn't seem to be declared anywhere in the libraries. It is used 2 times in `stm32fxxx_hal_libraries/00-STM32_LIBRARIES/tm_stm32_i2c_dma.c`.

Cheers

**Comment by MaJerle** [comment]
Correct. Thing is that this lib is not in working state. Never will be probably.

**Comment by jaguarondi** [comment]
I was trying to compile your library in SW4STM32. And in Eclipse all source files get compiled. There were a couple of other files/libs I had to delete before it would compile. I'll see if I can add them back in some way and contribute back.

Great library by the way!
